apps/arweave/src/ar.erl

```diff
@@ -541,6 +547,14 @@ parse_cli_args(["cm_exit_peer", Peer | Rest], C) ->
 			io:format("Peer ~p is invalid.~n", [Peer]),
 			parse_cli_args(Rest, C)
 	end;
+parse_cli_args(["pool_server" | Rest], C) ->
+	parse_cli_args(Rest, C#config{ pool_server = true });
+parse_cli_args(["pool_client" | Rest], C) ->
```

Comment:
Is pool_client only set when the current node is part of a CM set and is uing the CM exit node to communicate with the pool?

Reply:
ah wait, I see these are true/false values and not IP addresses. Got it.